jest.setup.js

```diff
@@ -0,0 +1,20 @@
+import '@testing-library/jest-dom/extend-expect';
+import { expect } from '@jest/globals';
+
+const toHaveLoader = (obj) => {
+  if (obj.queryByRole('progressbar')) {
+    return {
+      message: () => 'Must have loader with role "progressbar"',
+      pass: true
+    };
+  } else {
+    return {
+      message: () => 'Must have loader with role "progressbar"',
+      pass: false
+    };
+  }
+};
+
+expect.extend({
+  toHaveLoader
+});
```

Comment:
Ok so probably ok to just remove this file?

Reply:
You don't necessarily have to wait - this would just have a strict dependency on TEAMNADO-4245 which is okay (its in the release process now).

You can either leave the files here or merge TEAMNADO-4245 into this branch. The merge is preferred but it may be complicated now that these changes are already committed to this branch. 

I was mainly making this comment for future reference. When we have a branch that depends on code changes in another branch merging to get those changes is better to keep the history clean and accurately reflect where changes came from.

Reply:
Ok I see ill probably just leave file here as the changes and all have already been implemented to eliminate complication.